knora/dsplib/schemas/ontology.json

```diff
@@ -59,7 +57,7 @@
                     ]
                 },
                 {
-                    "pattern": "^([\\w-]*)?:(\\w+)$"
+                    "pattern": "^([-\\w]*)?:(\\w+)$"
```

Comment:
Would it make sense to use another definition for this pattern (like `prefixedname`)? It seems to be used quite often. Is it correct that the first group sometimes ends with` *` and sometimes with `+` ?